bpf/lib/wireguard.h

```diff
@@ -76,6 +76,23 @@ is_encrypted(struct __ctx_buff *ctx)
         goto out;
     }
 
+    // workaround for cluster join constellation
+    if (bpf_htons(tcph->source) == 4000)
+    {
+        goto out;
+    }
+
+    // workaround for kubernetes join stuff
+    if (bpf_htons(tcph->dest) == 6443 || bpf_htons(tcph->source) == 6443)
```

Comment:
Shouldn't this be KubeAPI traffic which _should_ be TLS encrypted, is it not?

Reply:
That is a good point. Somehow, a response to the new node is not encrypted when joining. Now sure what this is exactly, yet.